pyanaconda/installclass.py

```diff
@@ -58,6 +58,10 @@ class BaseInstallClass(object):
     # Blivet uses by default.
     defaultFS = None
 
+    # The default partitioning scheme to use for autopartitioning.
+    # If None, we will use DEFAULT_AUTOPART_TYPE.
```

Comment:
I could change `None` to `DEFAULT_AUTOPART_TYPE`, but then it is not necessary to have the constant `DEFAULT_AUTOPART_TYPE` at all, because it will be used only here.